### DISCUSSION TITLE
Anywhere to buy pre-made?

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#38] [Q&A] [answered]
I'm looking for a 4 port Gamecube adaptor to bluetooth to use my 8bitdo pads with - looks like this can solve...
I'm not convinced I have the skills to build this myself tho...

Can I buy one anywhere? Struggling to find...

**Comment by darthcloud** [comment]
~~I will have a few to sell in the near future, probably July but I've been pushing that date for the last 6 month. I'm in the process of moving so I'm very busy.~~

BTW if anyone want to step in and build some, feel free to do so :)

**Reply by vaderag** [reply]
Awesome! Any way to get 'on a list'?!
Not ruling out that I might try myself in the meantime...

**Reply by darthcloud** [reply]
Here: https://forms.gle/Kax3ys27aKUHJwde8

**Reply by vaderag** [reply]
To confirm - the "BlueRetro DevKit" item is the ESp32 + DB25 soldered / flashed / cased? Or something else?

**Reply by darthcloud** [reply]
Yes but not the case, so ESp32 + DB25 soldered / flashed. 

For case you got to buy a RPI4 case on your own.

In any case I will email to confirm when time comes.

**Comment by alastairbdoggett** [comment]
hey! similar situation for me -- absolute beginner wrt electronics + hardware and don't trust myself to attempt this. anyone willing to build a GC adaptor as a commission piece?

**Reply by darthcloud** [reply]
I really want to sell some but I'm in the middle of house moving, so probably sometimes around November from me.

This isn't a GC version but [Will's Console Modifications](https://twitter.com/WillConsole?s=20) will soon sell standalone PSX version.

**Reply by alastairbdoggett** [reply]
ok sounds good! thanks for the update and the link to the PSX version! will definitely be interested in grabbing one when you have some to sell, hope the move goes well :)

**Comment by jaysonlarose** [comment]
It'd be awesome if there was a storefront for ordering these!
I'd like to order one for my NES, one for my SNES, and one for my Genesis, plus one "just in case", along with the appropriate cables...

**Reply by pmgducati** [reply]
@jaysonlarose  I'm happy to help out getting you setup. I created a set of PCBs for almost all the systems blueretro supports. you can message me at my username at gmail. Just FYI I have no official shop and don't plan to either, but if someone needs help and a looking for few one offs I can handle that.

https://github.com/pmgducati/Blue-Retro-AIO-Units

**Comment by darthcloud** [comment, marked as answer]
As an update I do not plan to build any BlueRetro HW myself anymore.  I just don't have the time for selling HW.

I will let the community handle this need.

Check the repo [README](https://github.com/darthcloud/BlueRetro) for commercial solution.

For DIY you may want to look at @pmgducati PCBs here https://github.com/pmgducati/Blue-Retro-AIO-Units

You can take the gerber zip and upload it to any PCB manufacturer to get them.

**Comment by vicsidious** [comment]
[censored]

**Reply by darthcloud** [reply]
Please do not post link to that company in my repo. The are using the project branding in violation of the project License.

The project license do allow commercial use so I'm fine with nearly all available HW currently available.

But those specific guys crossed the line and I do not support them in anyway.

**Reply by vicsidious** [reply]
So sorry to hear that, I was in no way aware of the situation. My apologies. 
Since they did credit you, I thought it was legitimate.
